plebnet/clone/create-child.sh

```diff
@@ -29,7 +29,9 @@ export DEBIAN_FRONTEND=noninteractive
 cd
 
 CHILD_DNA_FILE=~/.config/Child_DNA.json
+#CHILD_QTABLE_FILE=
```

Comment:
@HusainKapadia why is this commented?

Reply:
fixed properly

Reply:
Because when I wrote this the QTable.json had not been created. I'll add the correct paths if you haven't done so yet